kubeconfig_education-eks-YBEeiURS

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+preferences: {}
+kind: Config
+
+clusters:
+- cluster:
+    server: https://D4A69E621E273C7ACCE3E12003FEF15D.gr7.eu-west-2.eks.amazonaws.com
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5RENDQWJDZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeE1ERXpNVEV6TXpFd01Wb1hEVE14TURFeU9URXpNekV3TVZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTWhCCkRQSW5icjR2bXBpVlJvOEo5WnJpOGNQUnJjZDdBZENmU3FJUmplRFdYbzF4RTZGcDRxcFAzRUdBQWZZWTFEakgKcFBjeEo1anQzWXFLbk9tSE03TGJZbXZURE5GYW5qZ0NwVk9pUzF0Qlh1TW9mRy9tS0JrbWVId1FXRDRUWndJcgpmd3JHU3pJYjlPVTdSYW5obzZQblQ1bTh4YkxCejNYWUZyNGxWQ0RjNDgyVEFKWHZCSVE5NDhwb3lpRWNlcnN5ClUxUFBJMlBvZENpaDFXRExBeUk1QTJEL0o5ZGtscVBka1RPNktGaGc3ZjZscEp4d2pTanV5RThxajdwZFJZL2QKYnVPTndOL2xaL0cySmNWQ0gwVzg2ekw3RnNyNno2RHZjbUpKRTRKaXN5ekVDcFBlMm1nTGVaNDd5S01qUWVqYwp3ejRiOXkySHJzRThGS2JXempVQ0F3RUFBYU1qTUNFd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFKSkVYcUFESk8rL1JpUWphK1RKOXhNZDRiUFAKSUtHYVVGdVF3UEhsSlQrWDRPVkVwM2lJS1FNYm8wRkQvNnRYWWhFS2VoMVUvVmQ2d0RFUXNuTC9YTEkxaXZwNApRcXBjTmJJeHNGa1o0c0VlVGxnSDNzdXdhZ2JuR0hTNE5rOFBxNmlzNVV1UFJqSFB0SWY1bHBCVnc3eVNaUmtmCjM1dmg0ZVltMFhRcHNUa2J5K3VmUzNYcWhRWFA5Z2JkeDdwdklRaFFJTzFHay95ZGY2T1hIcyt2SVJaNU1yemMKU3dFejRsbEN5KzFia3NFVVN6ZEFmU29BQUNMMU5FK1ZucjJvVHQyWWJZbTIrSm1wcFQ3aHk4ZGhUOS9aNGUzaApSZ0RKbkVsTXlOSEMwaXFnbW5GOWg5WlN3b2pRczIvWERlM3hzVVprMXVWN3drRklrdmFCbGhsQmpMUT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
A silly question just to be clear - this is not sensitive, is it?

Reply:
It is, it's just encoded with base64 tool